op-batcher/batcher/driver.go

```diff
@@ -514,7 +527,7 @@ func (l *BatchSubmitter) throttlingLoop(ctx context.Context) {
 			l.Log.Error("Can't reach sequencer execution RPC", "err", err)
 			return
 		}
-		pendingBytes := l.state.PendingDABytes()
+		pendingBytes := l.channelMgr.PendingDABytes()
```

Comment:
It may be fine not to guard this call. But strictly speaking we'd need to... at least an `RLock()`.

Reply:
If it's the _only_ instance of a read, it might not be worth making it a read only lock.